packages/dds/tree/src/test/feature-libraries/sequence-field/utils.ts

```diff
@@ -68,8 +68,9 @@ import { rebaseRevisionMetadataFromInfo } from "../../../feature-libraries/modul
 import { TestChangeset } from "./testEdits.js";
 
 export function assertChangesetsEqual<T>(actual: SF.Changeset<T>, expected: SF.Changeset<T>): void {
+	const updatedActual = purgeUnusedCellOrderingInfo(actual);
```

Comment:
This was deliberately omited before: the production code should not add/produce unused cell ordering info.
What scenario is this needed for?